<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSensor</class>
 <widget class="QMainWindow" name="QSensor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSensor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/icon.png</normaloff>assets/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>987</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutQt"/>
    <addaction name="actionLicense"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionExport_2"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open Tracking File</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export as png</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close FastAnalyzer</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
  <action name="loadWorkspaceAction">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="saveWorkspaceAction">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="closeWorkspaceAction">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>&amp;Reset</string>
   </property>
  </action>
  <action name="actionExport_2">
   <property name="text">
    <string>&amp;Export</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>QSensor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
